### ISSUE TITLE
TypeScript Declarations

### ISSUE THREAD

**Issue by YewoMhango** [JSLancerTeam/react-google-flight-datepicker#60]
There are no existing type declarations for the package on DefinitelyTyped, so it complicates trying to use the package with Typescript projects. I can also try to help out with achieving that if necessary (though I'm not particularly experienced with contributing to Open-source projects as of now)

**Comment by YewoMhango** [comment]
Hi @davidtran. I've managed to create a version of this project converted to typescript with just type annotations added, and no changes in functionality. Is it possible to upgrade this repo to a Typescript project entirely? If so, then I could create a pull request for that

**Comment by davidtran** [comment]
hi @YewoMhango sure, please create a PR for it. I will help you review it.

**Comment by arrrrrrrr** [comment]
Hi @YewoMhango ，will you provide TypeScript support？

**Comment by YewoMhango** [comment]
> Hi @YewoMhango ，will you provide TypeScript support？

Yes, hi there @arrrrrrrr . Somehow I didn't receive or see the notification for @davidtran 's reply to my request, such that I thought he might have not replied to it. But I guess I'll do it now, then

**Comment by shadoath** [comment]
@YewoMhango any update on this?
Or @davidtran are you planning to make this change?

**Comment by Telsho** [comment]
I'll try to see if I have time to switch it to ts, you are welcome to do PR with the latest changes @YewoMhango

**Comment by Telsho** [comment]
The codebase is now in Typescript, I close the issue.
